stack/README.md

```diff
@@ -15,4 +19,16 @@ docker compose up -d
 docker compose --profile frontend up -d
 ```
 
-additional configuration for the services is done via `*.env`-Files
\ No newline at end of file
+additional configuration for the services is done via `*.env`-Files
+
+### Keycloak
+
+An admin user and a test user ist created by `init-keycloak`. You can change the configuration via the keycloak ui.
```

Comment:
*ist* should probably be *are* here

Reply:
resolved with 3326e69820529c51b5056c4a83914b9959155115